Trell/Rpc/ToEngine.cs

```diff
@@ -90,9 +90,9 @@ public static EngineWrapper.Work.ArgType ToFunctionArg(this Function fn, EngineW
                 engine.CreateJsFile(fn.OnUpload.Filename, fn.OnUpload.Type, fn.OnUpload.Content.ToByteArray())
             ),
             Function.ValueOneofCase.Dynamic =>
-                fn.Dynamic.Data is null
+                fn.Dynamic.Arguments is null
                   ? EngineWrapper.Work.ArgType.NONE
-                  : new EngineWrapper.Work.ArgType.Json("arg", fn.Dynamic.Data.Text),
+                  : new EngineWrapper.Work.ArgType.Json("argv", fn.Dynamic.Arguments.ToString()),
```

Comment:
I lean toward converting this to
`EngineWrapper.Work.ArgType.Raw("argv", engine.CreateStringArray(fn.Dynamic.Arguments))`

And then adding a `IScriptObject CreateStringArray(IReadOnlyList<string> values)` to EngineWrapper. 

That would let us (in another PR) remove the ArgType.Json, I think. We could probably remove the ArgType.None as well and just use Raw everywhere.

My only caveat here is, 1. does it matter? and 2. how does the performance differ?